tests/resources/conformance_suites/nl_nt16/index.xml

```diff
@@ -12,6 +12,8 @@
     <testcase uri='br_kvk/4-12-testcase.xml' />
     <testcase uri='br_kvk/4-16-testcase.xml' />
     <testcase uri='fr_nl/1-01-testcase.xml' />
+    <testcase uri='fr_nl/1-03-testcase.xml' />
+    <testcase uri='fr_nl/1-05-testcase.xml' />
```

Comment:
Whoops, I should have caught this on the last PR during QA.

Reply:
I caught it right after you merged it.